### DISCUSSION TITLE
possible timeouts

### DISCUSSION THREAD

**Discussion by vitaliypavlenko** [nodejs/undici#2214] [Q&A] [answered]
what are the possible timeouts I can configure in communication? I am asking mostly about request timeouts, like timeout to open the connection, timeout to receive the response, timeout to read the response. I see general timeout was implemented but I do not see its support now: https://github.com/nodejs/undici/commit/6a934c479a8cc5fa0295ab7434329db853080a06

**Comment by metcoder95** [comment, marked as answer]
Check out: https://undici.nodejs.org/#/docs/api/Client?id=parameter-clientoptions
